ncm-network/src/main/perl/nmstate.pm

```diff
@@ -583,6 +610,21 @@ sub Configure
         $self->ethtool_opts_keeps_state($file_name, $ifacename, $iface, $exifiles);
     }
 
+    # configure vips defined under path /system/network/vips/ as dummy interface
+    # only if manage_vips is set to true.
+    if ($net->{manage_vips} && defined($net->{vips})) {
+        $self->verbose("Service address vips found, configuring dummy interfaces");
+        my $vips = $net->{vips};
+        my $idx = 0;
+        foreach my $name (sort keys %$vips) {
+            my $dummy_name="dummy$idx";
```

Comment:
spaces around `=`

but euhm, all dummy vip names might changes by adding a single interface with name that is earlier in alphabetic order? (i have no easy fix besides chagin the schema to a list, but just be aware of this "feature")